mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
+  //This function returns a random number between 0 and 9.
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
+  //This function joins two words together. 
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return word.concat(firstWord, secondWord, thirdWord);
```

Comment:
I don't think this will work - what is `word`?

The concat function is called on a string. You can solve this exercise with this function, but it will look (something) like this:

`return firstWord.concat(" ", secondWord, " ", thirdWord);`

Better to use interpolation: 
```
return `${firstWord} ${secondWord} ${thirdWord}`;